.github/workflows/cargo-build.yml

```diff
@@ -38,16 +38,6 @@ jobs:
       - name: Check out
         uses: actions/checkout@v3
 
-      - name: Install Rust
```

Comment:
Does this mean all the workflow files will start using the toolchain defined in the root (in the toolchain file)?

Reply:
not all, the ones define another toolchain, with override, that version will be used.